Packs/QuestKace/Integrations/QuestKace/QuestKace.yml

```diff
@@ -0,0 +1,549 @@
+    - default: false
+      description: summary of the new ticket.
+      isArray: false
+      name: summary
+      required: false
+      secret: false
+    - default: false
+      description: name of the impact of the new ticket.
+      isArray: false
+      name: impact
+      required: false
+      secret: false
+    - default: false
+      description: category name of the new ticket.
+      isArray: false
+      name: category
+      required: false
+      secret: false
+    - default: false
+      description: status name fo the new ticket.
+      isArray: false
+      name: status
+      required: false
+      secret: false
+    - default: false
+      description: priority name of the new ticket.
+      isArray: false
+      name: priority
+      required: false
+      secret: false
+    - default: false
+      description: name of machine of the new ticket.
+      isArray: false
+      name: machine
+      required: false
+      secret: false
+    - default: false
+      description: name of the asset of the new ticket.
+      isArray: false
+      name: asset
+      required: false
+      secret: false
+    - default: false
+      description: Can enter custom fields for the new ticket.
+      isArray: false
+      name: custom_fields
+      required: false
+      secret: false
+    - default: false
+      description: Queue in which the new ticket should be related to.
+      isArray: false
+      name: queue_id
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a new ticket to the system.
+    execution: false
+    name: kace-ticket-create
+  - arguments:
+    - default: false
+      description: updated title of the ticket.
+      isArray: false
+      name: title
+      required: false
+      secret: false
+    - default: false
+      description: updated summary for the ticket.
+      isArray: false
+      name: summary
+      required: false
+      secret: false
+    - default: false
+      description: updated impact of the ticket.
+      isArray: false
+      name: impact
+      required: false
+      secret: false
+    - default: false
+      description: updated category of the ticket.
+      isArray: false
+      name: category
+      required: false
+      secret: false
+    - default: false
+      description: updated status of the ticket.
+      isArray: false
+      name: status
+      required: false
+      secret: false
+    - default: false
+      description: updated priority of the ticket.
+      isArray: false
+      name: priority
+      required: false
+      secret: false
+    - default: false
+      description: updated machine of the ticket.
+      isArray: false
+      name: machine
+      required: false
+      secret: false
+    - default: false
+      description: updated asset of the ticket.
+      isArray: false
+      name: asset
+      required: false
+      secret: false
+    - default: false
+      description: custom fields for update of ticket.
```

Comment:
As we talked, let's make sure we have a more user friendly manner to use both OOTB fields and custom_fields.